src/query/storages/fuse/src/operations/commit.rs

```diff
@@ -269,8 +273,37 @@ impl FuseTable {
     ) -> Result<TableSnapshot> {
         // 1. merge stats with previous snapshot, if any
         let stats = if let Some(snapshot) = &previous {
-            let summary = &snapshot.summary;
-            merge_statistics(&statistics, summary)?
+            let mut summary = snapshot.summary.clone();
+            let mut fill_default_values = false;
+            for column_id in statistics.col_stats.keys() {
+                if !summary.col_stats.contains_key(column_id) {
+                    fill_default_values = true;
+                    break;
+                }
+            }
+            if fill_default_values {
+                let mut default_values = Vec::with_capacity(schema.num_fields());
+                for field in schema.fields() {
+                    default_values.push(field_default_value(ctx.clone(), field)?);
+                }
+                let leaf_default_values = schema.field_leaf_default_values(&default_values);
+                leaf_default_values
+                    .iter()
+                    .for_each(|(col_id, default_value)| {
+                        if !summary.col_stats.contains_key(col_id) {
+                            let col_stat = ColumnStatistics {
+                                min: default_value.to_owned(),
+                                max: default_value.to_owned(),
+                                null_count: summary.row_count,
```

Comment:
Need to check if default value is null. if null then null_count=summary.row_count,distinct_of_values=Some(0).
If non-null, null_count=0, distinct_of_values=Some(1).